e2e/cypress/integration/notifications/not_unless_Im_mentioned_spec.js

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @notifications
+
+function setReplyNotificationsSetting(idToToggle) {
+    // Navigate to settings modal
+    cy.toAccountSettingsModal();
+
+    // Select "Notifications"
+    cy.get('#notificationsButton').click();
+
+    // Notifications header should be visible
+    cy.get('#notificationSettingsTitle').
+        scrollIntoView().
+        should('be.visible').
+        and('contain', 'Notifications');
+
+    // Open up 'Reply Notifications' sub-section
+    cy.get('#commentsTitle').
+        scrollIntoView().
+        click();
+
+    cy.get(idToToggle).check().should('be.checked');
+
+    // Click “Save” and close modal
+    cy.get('#saveSetting').
+        scrollIntoView().
+        click();
+    cy.get('#accountSettingsHeader > .close').
+        click().
+        should('not.exist');
+
+    // Setup notification spy
+    cy.window().then((win) => {
+        function Notification(title, opts) {
+            this.title = title;
+            this.opts = opts;
+        }
+
+        Notification.requestPermission = function() {
+            return 'granted';
+        };
+
+        Notification.close = function() {
+            return true;
+        };
+
+        win.Notification = Notification;
+
+        cy.spy(win, 'Notification').as('notifySpy');
+    });
+
+    // Verify that we now have a Notification property
+    cy.window().should('have.property', 'Notification');
+}
+
+describe('reply-notifications', () => {
+    function ignoreUncaughtException() {
+        cy.on('uncaught:exception', (err) => {
+            expect(err.message).to.include('.close is not a function');
+
+            return false;
+        });
+    }
```

Comment:
Same here, please extract to `cypress/integration/support/notification.js`. I think we missed to export it before but I'm requesting to make it happen here.  Thanks.